src/angular/table/table/table.ts

```diff
@@ -57,12 +57,32 @@ export class SbbRecycleRows {}
   // element in the table. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption
   // We can't reuse `CDK_TABLE_TEMPLATE` because it's incompatible with local compilation mode.
   template: `
-    <ng-content select="caption"></ng-content>
-    <ng-content select="colgroup, col"></ng-content>
-    <ng-container headerRowOutlet></ng-container>
-    <ng-container rowOutlet></ng-container>
-    <ng-container noDataRowOutlet></ng-container>
-    <ng-container footerRowOutlet></ng-container>
+    <ng-content select="caption" />
+    <ng-content select="colgroup, col" />
+    <!--
+        Unprojected content throws a hydration error so we need this to capture it.
+        It gets removed on the client so it doesn't affect the layout.
+      -->
+    @if (_isServer) {
+      <ng-content />
+    }
+    @if (_isNativeHtmlTable) {
+      <thead role="rowgroup">
+        <ng-container headerRowOutlet />
+      </thead>
+      <tbody class="mdc-data-table__content" role="rowgroup">
```

Comment:
does this css class has any effect?

Reply:
Thanks, I copied this class unintentionally and removed it again.